Requirement already satisfied: google-api-python-client in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (2.88.0)
Requirement already satisfied: google-auth-httplib2>=0.1.0 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from google-api-python-client) (0.1.0)
Requirement already satisfied: google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from google-api-python-client) (2.11.0)
Requirement already satisfied: httplib2<1dev,>=0.15.0 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from google-api-python-client) (0.22.0)
Requirement already satisfied: uritemplate<5,>=3.0.1 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from google-api-python-client) (4.1.1)
Requirement already satisfied: google-auth<3.0.0dev,>=1.19.0 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from google-api-python-client) (2.19.0)
Requirement already satisfied: protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-api-python-client) (4.23.2)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.56.2 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-api-python-client) (1.59.0)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-api-python-client) (2.29.0)
Requirement already satisfied: urllib3<2.0 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from google-auth<3.0.0dev,>=1.19.0->google-api-python-client) (1.26.15)
Requirement already satisfied: six>=1.9.0 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from google-auth<3.0.0dev,>=1.19.0->google-api-python-client) (1.16.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in c:\users\sande\appdata\roaming\python\python38\site-packages (from google-auth<3.0.0dev,>=1.19.0->google-api-python-client) (0.3.0)
Requirement already satisfied: rsa<5,>=3.1.4 in c:\users\sande\appdata\roaming\python\python38\site-packages (from google-auth<3.0.0dev,>=1.19.0->google-api-python-client) (4.9)
Requirement already satisfied: cachetools<6.0,>=2.0.0 in c:\users\sande\appdata\roaming\python\python38\site-packages (from google-auth<3.0.0dev,>=1.19.0->google-api-python-client) (4.2.4)
Requirement already satisfied: pyparsing!=3.0.0,!=3.0.1,!=3.0.2,!=3.0.3,<4,>=2.4.2 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from httplib2<1dev,>=0.15.0->google-api-python-client) (3.0.9)
Requirement already satisfied: pyasn1<0.6.0,>=0.4.6 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from pyasn1-modules>=0.2.1->google-auth<3.0.0dev,>=1.19.0->google-api-python-client) (0.5.0)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-api-python-client) (2023.5.7)
Requirement already satisfied: idna<4,>=2.5 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-api-python-client) (3.4)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\sande\anaconda3\envs\ml2303_3_8_16\lib\site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-api-python-client) (2.0.4)
